README.md

```diff
@@ -1,4 +1,39 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
```

Comment:
Capital letters for headers?

Reply:
nope. Shall we? I assume capital letters if something is big/mature. And here is a small project, so small letters ;)